### DISCUSSION TITLE
[court] 사용자가 날짜, 시간, 바운더리로 농구장 검색하기

### DISCUSSION THREAD

**Discussion by sds1vrk** [slamapp/server#31] [Q&A]
1. 문제점 - 시간을 30분 단위로 나누고 예약된 max값을 찾는것이 어려움
2. 이유 - 코트 별로 id값이 다르고 stratTime과 endTime이 들어가기 때문에 그 사이의 값을 찾는 것이 어렵기 때문
3. 현재 나온 아이디어 - 30분 단위로 테이블을 쪼갠 후 join 
 - 문제점은 코트가 200개라면 200*48=최대 9600개의 row가 하루에 들어갈수 있어서 안됨. 

https://www.notion.so/backend-devcourse/GET-api-v1-courts-params-a640843e971544ac8c617f84dee848ad

Help me

**Comment by KwonYeKyeong** [comment]
헤이 그 때 한나랑 얘기나눴던 내용 여기에 공유해주시면 좋을꺼같아요~~~!!!
어떻게 진행하고있나요??
